test/integration/scheduler_perf/config/event_handling/add-node-unsched-pod-nodeunschedulable.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  generateName: pod-unsched-
+spec:
+  tolerations:
+  - key: allow
```

Comment:
Wouldn't this toleration actually allow the pod to schedule immediately?

Reply:
No, pod is still blocked by NodeUnschedulable plugin, because node has `unschedulable: true` set.